frontend/src/Home.js

```diff
@@ -5,17 +5,17 @@ import styled from '@emotion/styled'
 import { PhaseBanner } from './components/phase-banner'
 import { PageNotFound } from './PageNotFound'
 import { FileUploadPage } from './FileUploadPage'
-import { Suspectinfo } from './Suspectinfo'
+import { Suspectinfo } from './SuspectInfoYesNoPage'
```

Comment:
Did you start by branching off the latest master? ie
```
git checkout master
git pull
git checkout -b resetfont
```
The `SuspectInfoYesNoPage` is already in master.